src/harness/fakedatabase/pal/pal_db.json

```diff
@@ -0,0 +1,64 @@
+[
```

Comment:
This seems like the file that should be auto-generated as part of the default config. Why do you want to check it in?

I've made some comments below on the data contents/format which will apply in either case.

Reply:
this is an old file asked to be removed. We removed this file for now.